lib/tasks/redis.rake

```diff
@@ -0,0 +1,30 @@
+require "redis"
+
+namespace :redis do
+  desc "Make redis a slave of production redis"
+  task :make_slave => :environment do
+    raw_prod_redis_uri = ENV['PRODUCTION_REDIS_URL']
+    next if raw_prod_redis_uri.blank?
+
+    prod_redis_uri = URI.parse(raw_prod_redis_uri)
+    redis = Redis.new(url: ENV.fetch('REDIS_URL'), :driver => :hiredis)
+    redis.slaveof(prod_redis_uri.host, prod_redis_uri.port)
+    redis.config('set', 'masterauth', prod_redis_uri.password)
+    redis.config('set', 'slave-read-only', 'no')
+  end
+
+  desc "One-time copy of production redis"
+  task :copy_prod => :environment do
+    prod_redis_uri = ENV['PRODUCTION_REDIS_URL']
```

Comment:
This might need to also be `ENV.fetch('PRODUCTION_REDIS_URL')`, otherwise `prod_redis_uri` is `nil` and we'd call `redisSrc = Redis.new :url => nil` which is a valid line and returns the localhost version of Redis (no idea what would happen there since there isn't a Redis installation on the Heroku instance)

Reply:
My bad, this is handled by line 20!